models/glif_cond.h

```diff
@@ -130,36 +130,36 @@ V_reset    double   Reset potential of the membrane in mV (GLIF 1 or GLIF 3)
 -------------------------------------------------------------------------------
 th_spike_add               double         Threshold addition following spike
```

Comment:
Same question about parameter names here. Do they require backticks?

Reply:
I think parameters in tables don't necessarily need them. If we want to add them, it would take a fair bit of effort because the tables need to be realigned.

Reply:
fine with me.